src/models/nftRewardTier.ts

```diff
@@ -7,15 +7,16 @@ export type NftRewardTier = {
   remainingSupply: number | undefined
   imageUrl: string // link to ipfs
   name: string
+  id?: number
   /* @deprecated - now derived from comparing contributionFloor between each */
-  tierRank?: number | undefined // cheapest tier is 1
+  tierRank?: number // cheapest tier is 1
   externalLink: string | undefined
   description: string | undefined
 }
 
 // How the reward tiers are stored in the contracts
 export type ContractNftRewardTier = {
```

Comment:
**suggestion** non blocking, was thinking it would be nice to name this type after the contact struct name, like JB721Tier or whatever it is.

Reply:
For sure, good call